pandas/tests/frame/test_operators.py

```diff
@@ -827,13 +830,13 @@ def test_boolean_comparison(self):
         result = df == b
         assert_frame_equal(result, expected)
 
-        result = df == l
-        assert_frame_equal(result, expected)
+        with pytest.raises(ValueError):
+            result = df == l
 
-        result = df == tup
-        assert_frame_equal(result, expected)
+        with pytest.raises(ValueError):
+            result = df == tup
 
-        result = df == b_r
+        result = df == b_r  # broadcasts like ndarray (GH#23000)
```

Comment:
Changed following #23000 (previously in this PR this comparison raised ValueError)

Reply:
comments on previous